feconf.py

```diff
@@ -68,6 +69,9 @@
 RULES_DESCRIPTIONS_FILE_PATH = os.path.join(
     os.getcwd(), 'extensions', 'interactions', 'rules.json')
 
+# The ID of the classifier to be used as a default string classifier.
+DEFAULT_STRING_CLASSIFIER = 'LDAStringClassifier'
```

Comment:
That's a good point! What should we call it? I think INTERACTION_CLASSIFIER_MAPPING sounds fine ...